tests/integration/commands/dev/dev-miscellaneous.test.ts

```diff
@@ -8,27 +8,36 @@ import execa from 'execa'
 import getAvailablePort from 'get-port'
 import jwt from 'jsonwebtoken'
 import fetch from 'node-fetch'
-import { describe, test } from 'vitest'
+import { type TestContext, type TaskContext, describe, test } from 'vitest'
+import type { HandlerContext } from '@netlify/functions'
```

Comment:
half of these are Functions Context (this type) and the other half are Edge Functions Context, which should be imported from `@netlify/edge-functions`